README.md

```diff
@@ -1,2 +1,7 @@
 # osdg-ranker
-OSDG Ranker. A ranking of people based on the amount of Open Source work they've done.
+OSDG Ranker. A ranking of people based on the amount of Open Source work they've done. This project will serve as a incentive for people to start open source and keep contributing to it. This project will fetch data from github with the help of git api and html scrappers, then stores the data in a database. Then the data is displayed onto the website(once it is ready) http://osdg.iiit.ac.in/. Currently the person's contribution in a repo would be evaluated by the ratio of  sum of lines he/she has changed in the repo and total number of lines in it. Finally this ratio will be multiplied by the value of the repo, which may be given by stars or the number of contributors. A persons total contribution be then hence the sum of his contributions in all the projects he has yet worked on. This would be then updated on a regularly basis.
```

Comment:
> amount of Open Source work they've done

Are we talking about Open Source contributions in general, or contributions to OSDG-IIITH?